src/components/membership-promo/more.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+// @twreporter
+import requestOrigins from '@twreporter/core/lib/constants/request-origins'
+import Link from '@twreporter/react-components/lib/customized-link'
+import { PillButton } from '@twreporter/react-components/lib/button'
+import {
+  BRANCH,
+  BRANCH_PROP_TYPES,
+} from '@twreporter/core/lib/constants/release-branch'
+
+const StretchPillButton = styled(PillButton)`
+  width: auto;
+  display: flex;
+  justify-content: center;
+`
+const MoreButton = ({ releaseBranch = BRANCH.master, ...props }) => (
+  <Link
+    to={`${requestOrigins.forClientSideRendering[releaseBranch].support}#roles`}
```

Comment:
i think the has should be `#plan`
<img width="162" alt="image" src="https://github.com/twreporter/twreporter-react/assets/61230075/f245495b-b16e-488f-8c1a-0ceebd557de9">

Reply:
You're right, updated. ([4ffa4f3](https://github.com/twreporter/twreporter-react/pull/2570/commits/4ffa4f39ff42e04ab56a20c1e6d6ad7f4e4460b4))